litex_boards/targets/sds1104xe.py

```diff
@@ -36,6 +36,8 @@
 
 from liteeth.phy.mii import LiteEthPHYMII
 
+from sds1104xe_offsetdac import OffsetDac
```

Comment:
Where do we put more complex per-board logic? This component can likely be shared on similar hardware (such as the SDS1202X-E, the 2-channel variant).